tests/src/test/java/stepDefinitions/frontendstepdefinitions/LandingPageTest.java

```diff
@@ -1,48 +1,88 @@
 package stepDefinitions.frontendstepdefinitions;
 
 import ca.bc.gov.open.jagefilingapi.qa.config.ReadConfig;
+import ca.bc.gov.open.jagefilingapi.qa.frontend.pages.EfilingPage;
 import ca.bc.gov.open.jagefilingapi.qa.frontend.pages.LandingPage;
 import ca.bc.gov.open.jagefilingapi.qa.frontendutils.DriverClass;
-import io.cucumber.java.en.And;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
 import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 import org.junit.Assert;
+
 import java.io.IOException;
 
 public class LandingPageTest extends DriverClass {
 
     ReadConfig readConfig;
     LandingPage landingPage;
+    EfilingPage eFilingPage;
 
-    @Given("browser is initialized")
-    public void browser_is_initialized() throws IOException {
+    @Before ("@frontend")
+    public void setUp() throws IOException {
         initializeDriver();
         log.info("Browser is initialized");
     }
 
-    @Given("user is navigated to the landing page")
-    public void user_is_navigated_to_the_landing_page() throws IOException {
+    @After ("@frontend")
+    public void tearDown() {
+        driver.quit();
+        log.info("Browser closed");
+    }
+
+    @Given("user is on the landing page")
+    public void user_is_on_the_landing_page() throws IOException {
         readConfig = new ReadConfig();
+        landingPage = new LandingPage(driver);
 
         String url = readConfig.getBaseUrl();
         driver.get(url);
         log.info("Page is launched");
+
+        String actualTitle = landingPage.verifyLandingPageTitle();
+        String expTitle = "eFiling Frontend Demo App";
```

Comment:
same comment about title here, let's make it more clear with naming 😄

Reply:
Changed page title in the html and test as well.